src/components/VCheckbox/VCheckbox.vue

```diff
@@ -1,17 +1,21 @@
 <template>
   <label :for="id" class="checkbox-label" :class="labelClasses">
+    <!--
+    The checkbox focus style is a slight variation on the `focus-slim-tx` style.
+    Because it becomes filled when checked, it also needs the
+    `checked:focus-visible:border-white` class.
+    -->
     <input
       :id="id"
       type="checkbox"
-      class="checkbox bg-white"
+      class="checkbox h-5 w-5 flex-shrink-0 appearance-none rounded-sm bg-white me-3 focus-slim-tx checked:border-dark-charcoal checked:bg-dark-charcoal checked:focus-visible:border-white disabled:border-dark-charcoal-40 disabled:bg-dark-charcoal-10 checked:disabled:border-dark-charcoal-40 checked:disabled:bg-dark-charcoal-40"
```

Comment:
I see that there are some snapshot changes: the border of the checkbox in resting state was light charcoal, and now it's darker. Not sure why as the border values seem not to have changed (dark-charcoal).

Reply:
The changes to the checkbox were intentional to differentiate the different states.

| Enabled | Checked | Hovered | Focused | Border | Background | Comment |
|-|-|-|-|-|-|-|
| | | | | dark-charcoal-40 | white | Resting state |
| | | | ✅ | transparent | white | Slim transparent style |
| | | ✅ | | dark-charcoal | white | Border darkens |
| | | ✅ | ✅ | transparent | white | Focus takes precedence |
| | ✅ | | | dark-charcoal | dark-charcoal | Dark charcoal fill |
| | ✅ | | ✅ | transparent | dark-charcoal | White stroke between focus & fill |
| | ✅ | ✅ | | dark-charcoal | dark-charcoal | Same as checked + resting |
| | ✅ | ✅ | ✅ | transparent | dark-charcoal | Same as checked + focused |
| ✅ | | | | dark-charcoal-40 | dark-charcoal-10 | Background darkens |
| ✅ | ✅ | | | dark-charcoal-40 | dark-charcoal-40 | Background matches border |

Disabled checkbox cannot be focused and shows no hover interaction.

The only changes to the checkbox were
- a lighter dark-charcoal-40 border which becomes dark-charcoal on hover
- the new focus style to match the WordPress DL

Reply:
Tagging @panchovm for advice.

Reply:
Updated the checkbox to remove the hover state in a4ded5f.